az_quickstart.info.yml

```diff
@@ -11,6 +11,7 @@ install:
   - az_cas
   - az_core
   - az_flexible_page
+  - az_demo
```

Comment:
Can `az_demo` be optional?

Reply:
Here is a reference page that didn't help me: https://www.drupal.org/docs/8/distributions/creating-distributions/how-to-write-a-drupal-8-installation-profile#s-infoyml-file

Reply:
I think this list in `az_quickstart.info.yml` isn't a required list of modules like in module dependencies, and only ones that are first installed by the installation profile. It seemed to allow me to disable the module (which rolls back the content when `hook_uninstall()` fires).

Is it preferred that the demo content be not enabled by default?

Reply:
Ok, really just wanted to be able to disable it,sorry.